src/pages/policies/policies.tsx

```diff
@@ -110,7 +110,7 @@ function PoliciesLayout(props: PoliciesLayoutProps) {
   const query = params.get('ipfs-hash') || '';
   const handleSubmit: FormEventHandler<HTMLFormElement> = (ev) => {
     ev.preventDefault();
-    const { value } = ev.currentTarget.query;
```

Comment:
Are we sure `query` is going to be defined and reading `value` will not produce an error?

Reply:
Yeah the form event has all field elements available on it, and because this input has a name prop/attribute, it is available via `currentTarget.query` or `currentTarget.elements.query`.  Hashtag use the platform 🙂